packages/playback/src/lib/types/configuration.declarations.ts

```diff
@@ -61,10 +62,120 @@ export interface PlayerHlsConfiguration {
   customTagMap: CustomTagMap;
   transformTagValue: TransformTagValue;
   transformTagAttributes: TransformTagAttributes;
+
+}
+
+/** Would be used if we allowed the user to pass in a persistent session */
+// If we need this, move it to another file
+// export interface KeySessionMetadata {
+//   sessionId: string;
+//   initData: Uint8Array;
+//   initDataType: string;
+// }
```

Comment:
we only need sessionId in this case, but we can go further and encapsulate this logic in eme-controller itself